connectivity/tests/host.go

```diff
@@ -88,21 +96,31 @@ func (s *podToHostPort) Run(ctx context.Context, t *check.Test) {
 		for _, echo := range ct.EchoPods() {
 			echo := echo // copy to avoid memory aliasing when using reference
 
-			baseURL := fmt.Sprintf("%s://%s:%d%s", echo.Scheme(), echo.Pod.Status.HostIP, check.EchoServerHostPort, echo.Path())
-			ep := check.HTTPEndpoint(echo.Name(), baseURL)
-			t.NewAction(s, fmt.Sprintf("curl-%d", i), &client, ep, check.IPFamilyAny).Run(func(a *check.Action) {
-				a.ExecInPod(ctx, ct.CurlCommand(ep, check.IPFamilyAny))
-
-				a.ValidateFlows(ctx, client, a.GetEgressRequirements(check.FlowParameters{
-					// Because the HostPort request is NATed, we might only
-					// observe flows after DNAT has been applied (e.g. by
-					// HostReachableServices),
-					AltDstIP:   echo.Address(check.IPFamilyAny),
-					AltDstPort: echo.Port(),
-				}))
-			})
+			var ipFamStr string
 
-			i++
+			t.ForEachIPFamily(func(ipFam check.IPFamily) {
```

Comment:
The change which adds `t.ForEachIPFamily(func(ipFam check.IPFamily)` here does not make sense. The code within the func  does not actually use a different IP (it still calls the CurlCommand with IPFamilyAny).

I would suggest do revert this change here and only modify tests which currently already do call `ForEachIPFamily` and use different IPs in each invocation.

Reply:
Makes sense. 
Okay so confirming once again, I need to revert the changes done in `podToHostPort` since it still calls the CurlCommand with IPFamilyAny.

Reply:
Correct!